src/gui/Note.cpp

```diff
@@ -55,7 +55,7 @@ void Note::deallocate() {
 	// Don't bother actually deleting the textures, we'll just need them again!
 	background = NULL, prevPage = NULL, nextPage = NULL;
 	
-	allocatedForRatio = Vec2f_ZERO;
+	currentScale = 1.0f;
```

Comment:
This should probably be 0 or it might not get allocated at all if bookScale == 1.